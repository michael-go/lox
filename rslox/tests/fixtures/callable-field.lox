class Oops {
  init() {
    fun f() {
      print "I'm not a method";
    }

    this.field = f;
  }
}

var oops = Oops();
oops.field();