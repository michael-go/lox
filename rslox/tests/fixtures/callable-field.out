# exit code: 0
# stdout:
"I'm not a method"

# stderr:

